Example/Swift/ViewController.swift

```diff
@@ -54,6 +61,15 @@ class ViewController: UIViewController, MGLMapViewDelegate, AVSpeechSynthesizerD
         getRoute()
     }
     
+    @IBAction func didToggleNavigation(_ sender: Any) {
```

Comment:
Since this is a demo, you could also use a UISwitch. That would eliminate some code (compared to UIButton), so the example can focus on what matters.